js/output/pjs/pjs-utils.js

```diff
@@ -1,4 +1,4 @@
-window.PJSUtils = {
+const PJSUtils = {
```

Comment:
I've identified three files in this diff that would be better done that way:
- Pjs-utils
- Pjs-ast-transforms
- Ast-builder

I'm happy to make that change in this diff, if you're OK with the larger diff. (Sometimes big whitespace changes throw off the diffing so the PR looks bigger, I don't know how it'll render that change). Those files are minimally changed otherwise.

Reply:
Sorry for using imprecise language... I meant "future PR" when I said "second pass".

Reply:
Ok, noting as a task for future PR.